src/C2_economic_functions.py

```diff
@@ -124,7 +128,7 @@ def fuel_price_present_value(economics,):
         economics.update({"price_fuel": fuel_price_i})
     else:
         for i in range(0, economics["project_lifetime"]):
-            cash_flow_fuel_l += fuel_price_i / (1 + economics["wacc"]) ** (i)
+            cash_flow_fuel_l += fuel_price_i / (1 + economics["discount_factor"]) ** (i)
```

Comment:
Can you search and replace all instances of the string "discount_factor" by DISCOUNTFACTOR (comp. [constants_json_strings.py](https://github.com/rl-institut/mvs_eland/blob/dev/src/constants_json_strings.py#L49), and then in the first lines where you import constants from that file add that constant as well?